scripts/k6/user-scenario-test.js

```
@@ -0,0 +1,51 @@
+        { duration: '10s', target: 200 },
+    ],
+    thresholds: {
+        'http_req_failed': ['rate<0.05'],            // 실패율 5% 미만으로 완화
+        'http_req_duration': ['p(95)<2000', 'avg<1000'], // 임계값 완화
+        'checks': ['rate>0.95'],                     // 체크 성공률 95% 이상으로 완화
+    },
+};
+
+export default function () {
+    // 1. 홈화면 조회
+    let homeRes = http.get('https://api.movie-bookie.shop/api/events/home', {
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'homeRes' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)